### DISCUSSION TITLE
left icon with button variant `unstyled` doesn't work properly

### DISCUSSION THREAD

**Discussion by VikasPatel** [chakra-ui/chakra-ui#7081] [Show and tell]
I tried to use the Left icon with the button variant `unstyled`, but doesn't look good. vertical alignment is not correct. Any other way I can handle it?

```
<Stack direction='row' spacing={4}>
  <Button leftIcon={<EmailIcon />} variant='unstyled'>
    Email
  </Button>
</Stack>
```

<img width="844" alt="Screenshot 2022-12-07 at 2 11 57 PM" src="https://user-images.githubusercontent.com/68289785/206130459-1350d286-2871-489d-92dd-3ad6b9b195ff.png">

**Comment by VikasPatel** [comment]
Found the solution. Have to give `display: inline-flex` to the button.
```
<Stack direction='row' spacing={4}>
  <Button leftIcon={<EmailIcon />} variant='unstyled' display="inline-flex">
    Email
  </Button>
</Stack>
```

**Comment by tomas223** [comment]
I'd say it's because is unstyled. I believe what you are looking for is `ghost` variant.

**Reply by VikasPatel** [reply]
@tomas223 In the `ghost` variant we get some hover effects which I did not want. I wanted `unstyled` with the left or right icon.
